src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -205,13 +206,102 @@ protected function setupAdmin()
         $em->persist($user);
 
         $config = new Config($user);
-        $config->setTheme($this->getContainer()->getParameter('theme'));
-        $config->setItemsPerPage($this->getContainer()->getParameter('items_on_page'));
-        $config->setRssLimit($this->getContainer()->getParameter('rss_limit'));
-        $config->setLanguage($this->getContainer()->getParameter('language'));
+        $config->setTheme($this->getContainer()->getParameter('wallabag_core.theme'));
+        $config->setItemsPerPage($this->getContainer()->getParameter('wallabag_core.items_on_page'));
+        $config->setRssLimit($this->getContainer()->getParameter('wallabag_core.rss_limit'));
+        $config->setLanguage($this->getContainer()->getParameter('wallabag_core.language'));
 
         $em->persist($config);
 
+        // cleanup before insert new stuff
+        $em->createQuery('DELETE FROM CraueConfigBundle:Setting')->execute();
+
+        $settings = [
```

Comment:
These could come from an external source, so that the installer doesn't have to modified each time, could it not ?